packages/terra-embedded-content-consumer/src/EmbeddedContentConsumer.scss

```diff
@@ -0,0 +1,19 @@
+/* stylelint-disable max-nesting-depth */
+/* stylelint-disable selector-class-pattern */
+/* stylelint-disable selector-max-compound-selectors */
+:local {
+  .fill {
+    height: 100%;
+
+    :global {
+      .xfc {
+        height: 100%;
+     
+        iframe {
+          height: 100%;
```

Comment:
`height: 100%` would possibly break the iframe's auto resizing feature, because the height of iframe should be changed dynamically based on the height of its embedded content. That's the default config. 
If user wants to change the default resizing logic, they can choose to pass in an additional param called `resizeConfig` to prop `options` (I made a comment above about the missing of resizeConfig in option's shape).